docs/get-must-not-be-lro.md

```diff
@@ -0,0 +1,70 @@
+# GetMustNotBeLRO
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-14
+
+## Output Message
+
+The Get call cannot be Long Running Operation and it must not have `x-ms-long-running-operation-options` property block.
+
+## Description
+
+Only asynchronous(i.e. Long Running Operation) can have `x-ms-long-running-operation-options` property.
+The Get calls are synchronous and it MUST NOT have
+    - `x-ms-long-running-operation-options` property block
+    - `x-ms-long-running-operation-options` set to `true`
+
+## How to fix the violation
+
+Ensure that the get operation `x-ms-long-running-operation` property set to `false` and MUST NOT contain `x-ms-long-running-operation-options` property block.
```

Comment:
>get [](http://example.com/codeflow?start=16&length=3)

can we maintain consistency with the verb and refer it as GET everywhere in the doc

Reply:
done